### PR TITLE
Fix | PEO-590 | [CircleCi] project configuration

### PR DESCRIPTION
**Pull request by kolyanjy**
Jira - https://garage.atlassian.net/browse/PEO-590

### CODE REVIEW

File: intro_to_circleci.md [hunk 1]
```diff
@@ -459,19 +445,10 @@ By default, CircleCI builds every commit from every branch. This behavior may be
...
-By default, CircleCI does not build PRs from forked repositories. To change this setting, go to the **Advanced Settings** of your project and set the **Build forked pull requests** option to On.
-
-### Caching
-Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub repository-id of the originator of the fork PR to identify the cache.
-
-- PRs from the same fork repo will share a cache (this includes, as previously stated, that PRs in the master repo share a cache with master).
-
-- Two PRs in different Fork Repos will have different caches.
+### Auto-cancel redundant builds
+With the exception of your default branch, we will automatically cancel any queued or running builds on a branch when a newer build is triggered on that same branch. Scheduled workflows and re-runs are not auto-canceled.
```
Comment (D3N) [inline intro_to_circleci.md]: Not sure about "we". Sounds strange. I would change it to this one:

With the exception of your default branch, any queued or running builds on a branch will be automatically cancel when a newer build is triggered on that same branch. Scheduled workflows and re-runs are not auto-canceled.

File: intro_to_circleci.md [hunk 2]
```diff
@@ -434,22 +434,8 @@ jobs:
...
-
-**Note:** If you are building an open source project on macOS, contact billing@circleci.com to enable these additional containers.
-
-### Security
-While open source can be a liberating practice, take care not to liberate sensitive information.
-
-- If your repository is public, your CircleCI project and its build logs are also public. Pay attention to the information you choose to print.
-
-- Environment variables set in the CircleCI application are hidden from the public, these variables will not be shared in [forked PRs](https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests) unless explicitly enabled.
+To support the open source community, projects that are public on GitHub or Bitbucket receive three free build containers, for a total of four containers.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: WHERE is private?

File: intro_to_circleci.md [hunk 3]
```diff
@@ -459,19 +445,10 @@ By default, CircleCI builds every commit from every branch. This behavior may be
...
-Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub repository-id of the originator of the fork PR to identify the cache.
-
-- PRs from the same fork repo will share a cache (this includes, as previously stated, that PRs in the master repo share a cache with master).
-
-- Two PRs in different Fork Repos will have different caches.
+### Auto-cancel redundant builds
+With the exception of your default branch, any queued or running builds on a branch will be automatically cancel when a newer build is triggered on that same branch. Scheduled workflows and re-runs are not auto-canceled.
 
-Currently there is no pre-population of caches because this optimization hasn’t made it to the top of the priority list yet.
+Pipelines must be enabled in order to use this feature.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: Recommended configurations

**Comment by dzemlianoi-double** [comment]
Merge after fix

File: intro_to_circleci.md [hunk 4]
```diff
@@ -459,19 +445,17 @@ By default, CircleCI builds every commit from every branch. This behavior may be
...
 
-### Build Pull Requests From Forked Repositories
-Many open source projects accept PRs from forked repositories. Building these PRs is an effective way to catch bugs before manually reviewing changes.
-
-By default, CircleCI does not build PRs from forked repositories. To change this setting, go to the **Advanced Settings** of your project and set the **Build forked pull requests** option to On.
-
-### Caching
-Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub repository-id of the originator of the fork PR to identify the cache.
+### Auto-cancel redundant builds
+automatically closes the build if there is a **newer**.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: Automatically closes all old build for some branch/PR if there is a newer one

**Pull request merged**